### ISSUE TITLE
Component tries to connect with local IP address while remote has been given in setup

### ISSUE THREAD

**Issue by xzaz** [roleoroleo/yi-hack_ha_integration#85]
Situation:
My HA is hosted externally and I want to add the cam. Everything works fine until I add it to the dashboard. 

`2022-05-16 09:29:23 ERROR (stream_worker) [homeassistant.components.stream.stream.camera.yi_hack_a2_713b1a_cam] Error from stream worker: Error opening stream (ERRORTYPE_110, Operation timed out) rtsp://****:****@192.168.178.29/ch0_0.h264`

It tries to connect to the cam on the local IP address. 

So basically its this rule: 
https://github.com/roleoroleo/yi-hack_ha_integration/blob/main/custom_components/yi_hack/camera.py#L192
